<?xml version="1.0"?>
<launch>

  <arg name="Left_name" default="Left_wheel_data.csv"/>
  <arg name="Right_name" default="Right_wheel_data.csv"/>

<!-- Left Wheel -->
  <node name="PID_Left_wheel_output_plotter" pkg="two_wheel" type="PID_csv_outputter.py" output="screen">
    <param name="csv_path" value="$(find two_wheel)/csv/$(arg Left_name)"/>
    <remap from="/rpm_data" to="Left_wheel_rpm_data"/>
  </node>

  <node name="Left_Arduino_node" pkg="rosserial_python" type="serial_node.py">
    <param name="port" value="/dev/ttyS1"/>
  </node>

<!--Right Wheel-->
  <node name="PID_Right_wheel_output_plotter" pkg="two_wheel" type="PID_csv_outputter.py">
    <param name="csv_path" value="$(find two_wheel)/csv/$(arg Right_name)"/>
    <remap from="/rpm_data" to="Right_wheel_rpm_data"/>
  </node>

  <node name="Right_Arduino_node" pkg="rosserial_python" type="serial_node.py">
    <param name="port" value="/dev/ttyS1"/>
  </node>


</launch>
